dabplus-on-air-processing.spec

```diff
@@ -11,7 +11,7 @@ Name:     dabplus-on-air-processing
 BuildRequires: make
 BuildRequires: systemd
 
-Requires: liquidsoap
+Requires: liquidsoap = ${_ls_version}
```

Comment:
Liquidsoap seems to use [some sort of Semver](https://github.com/savonet/liquidsoap/blob/551b2acd46b3de6b48da7e4cd9f1f9cd370a3b8f/RELEASING#L28-L34). Is there a specific reason for strictly locking the dependency to an exact version, rather than to a minimum (`liquidsoap >= 1.3.0`) or a major (or at least minor) version (`liquidsoap >= 1.3.0, liquidsoap < 2.0.0`)? This would allow later liquidsoap bugfix updates without having to touch this spec file.